editoast/assets/signal_sprites/generate-atlas.sh

```diff
@@ -20,7 +20,8 @@ for signaling_system in $(ls); do
   cp "${tmp_dir}"/sprites/sprites* "${signaling_system}"
 
   # Add a linefeed to the json files
-  for json_file in $(ls  "${signaling_system}"/*.json); do
+  for json_file in "${signaling_system}"/*.json; do
+  [ -e "$json_file" ] || continue
```

Comment:
When does globbing expands to a path to a file that doesn't exist?

Reply:
When no file match it at all, it seems to return the given string

For example 
```bash
alice@juna:~/dev/osrd$ for test in nope*.json; do echo $test;   done
nope*.json
alice@juna:~/dev/osrd$ for test in ni*; do echo $test;   done
nix
```

Reply:
Ooooh, right! Bash is weird.

(note: the test could be indented in the loop body)

Thanks!

Reply:
Oops, fixed, thanks!